### ISSUE TITLE
As a content creator, I would like to enter/upload data and have it display as a chart

### ISSUE THREAD

**Issue by makurj** [uiowa/uiowa#5368]
# Story

As a content creator, I would like to enter data into fields or upload a CSV and have it display as pie charts so I don't have to upload a static image each time the data changes, and it's better for accessibility.
Example: https://tippie.uiowa.edu/iowa-mba/admissions/class-profile

![image](https://user-images.githubusercontent.com/52135878/173628214-be19a30c-321b-401b-8825-79eb52fdc34d.png)

**Comment by joewhitsitt** [comment]
Is this the same as https://github.com/uiowa/uiowa/issues/4787

**Comment by makurj** [comment]
#4787 is about pulling in from data feeds. This one is about uploading or enter data.

**Comment by joewhitsitt** [comment]
https://www.drupal.org/project/charts keeps gaining ground and the highcharts library is being recognized for their work towards being accessible https://www.highcharts.com/blog/accessibility/#exploreFeatures

**Comment by bspeare** [comment]
I just tested out `[5.0.8](https://www.drupal.org/project/charts/releases/5.0.8)` of Charts and it works great.   Highcharts is very keyboard operable compared to the others.   Would it be worth adding this module for admin use only and recommend it as an alternative to PowerBI and Tableau embeds when we get those requests?

**Comment by bspeare** [comment]
I think https://github.com/uiowa/uiowa/issues/6328 is going to need data visualization in the layouts and the charts module would help greatly with that as well.

**Comment by bspeare** [comment]
Here's what the Tippie donut chart above would look like in Highcharts.  We would definitely want to change the font size and family. 

<img width="769" alt="Screenshot 2023-10-19 at 1 22 43 PM" src="https://github.com/uiowa/uiowa/assets/1036433/e9d0baf6-cd67-4cc3-a632-9d7cb1c9baee">

**Comment by joewhitsitt** [comment]
How is it implemented? Block plugin? Would we need to add-in headline, unique id, lb_styles, etc.?

**Comment by bspeare** [comment]
Yes, there's a charts block that comes with the module.   We would need to add the headline field but the unique id shows up automatically on the block already.   `lb_styles` could be a later enhancement or we would just add the default "Remove top margin" style to begin?

**Comment by bspeare** [comment]
I've tested this module locally so many times that I went ahead and created a charts feature to save configuration time in the future at https://github.com/uiowa/uiowa/pull/7546.   The Demand Jump project is interested in using some charts on their pages for Journalism.

**Comment by bspeare** [comment]
A question from ITS:

> We were curious if there is a Drupal module available that other units have used for displaying org charts rather than uploading PDFs constantly. We worried that the PDFs are most likely not accessible the way they are currently and we’re working to remediate that, so in turn wondered if there would be a better solution out there that we could use when we’re needing to display these on our website. We’ve realized how frequently our org charts are getting updated and swapped out and additionally want to ensure that they’re all accessible. Just wanted to brainstorm if there were other Drupal options we could utilize and implement rather than adding more pdfs to the site.

**Comment by bspeare** [comment]
True, but https://www.highcharts.com/docs/chart-and-series-types/organization-chart is probably possible with the charts module.  Maybe we could design some sort of chart component using cards and `<ul>` as well.

**Comment by pyrello** [comment]
> > displaying org charts rather than uploading PDFs constantly
> 
> I feel like this is a different type of content that could just be provided as a nested UL.

I worked on an org chart solution for a previous job. It actually a fairly complex thing to solve for. In most organizations its too much data to show the whole org chart so you end up dynamically showing some portion of it and then having a way to navigate inside of it.

I'm not seeing any options in with the Charts module that would make it obviously a good fit for something like this. This seems like a different feature request to me.
